qutebrowser/browser/network/proxy.py

```diff
@@ -61,8 +61,7 @@ def queryProxy(self, query):
         """
         proxy = config.val.content.proxy
         if proxy is configtypes.SYSTEM_PROXY:
-            # On Linux, use "export http_proxy=socks5://host:port" to manually
-            # set system proxy.
+            # On Linux, use "export http_proxy=socks5://host:port" to manually set system proxy
```

Comment:
You're still reverting/changing things I did in the `master` branch here...

Reply:
Sorry, IIRC that was actually done BEFORE your change in master... how should I handle that?